lib/createReducer.js

```diff
@@ -17,6 +17,11 @@ export default (initialState, handlers) => {
     throw new Error('handlers must be an object')
   }
 
+  // handlers cannot have an undefined key
+  if (R.any(R.equals('undefined'))(R.keys(handlers))) {
```

Comment:
I wouldn't have believed you if i didn't see the tests.  It's a string when you run it through `keys` eh?